packages/client/src/components/CreatePost.js

```diff
@@ -15,7 +15,8 @@ import Footer from "./Footer";
 import axios from "axios";
 
 function CreatePost() {
-  const { user, isAdmin, getAdminStatus } = useContext(AuthContext);
+  const { user, isAdmin, getAdminStatus, logout } = useContext(AuthContext);
+  const [editSubmitted, setEditSubmitted] = useState(false);
```

Comment:
Edit submitted isn't being used on any of these?

Reply:
my mistake!